### PR TITLE
Fix isZero is undefined

### PR DESCRIPTION
**Pull request by rickycodes**
**Description**

Fixes:

<image width=300 src=https://user-images.githubusercontent.com/675259/117220447-3d1ecc80-add5-11eb-9a7d-7cf086385805.png />

**Checklist**

* [x] There is a related GitHub issue
* [x] Tests are included if applicable
* [x] Any added code is fully documented

**Issue**

Resolves #???

### CODE REVIEW

File: app/components/UI/Tokens/index.js [hunk 1]
```diff
@@ -266,7 +266,7 @@ class Tokens extends PureComponent {
 		const tokensToDisplay = hideZeroBalanceTokens
 			? tokens.filter(token => {
 					const { address, symbol } = token;
-					return (tokenBalances[address] && !tokenBalances[address].isZero()) || symbol === 'ETH';
+					return (tokenBalances[address] && !tokenBalances[address]?.isZero()) || symbol === 'ETH';
```
Comment (estebanmino) [inline app/components/UI/Tokens/index.js:269]: I had the same issue today but `isZero` was defined but not a method

```suggestion
					return (tokenBalances[address] && !tokenBalances[address]?.isZero?.()) || symbol === 'ETH';
```

**Comment by estebanmino** [comment]
this also happens here https://github.com/MetaMask/metamask-mobile/blob/develop/app/components/UI/Tokens/index.js#L249

**Comment by rickycodes** [comment]
@estebanmino should be good now

**Review by estebanmino — APPROVED** [review]
LGTM

**Pull request merged**